website/docs/reference/sources/journald.md.erb

```diff
@@ -34,7 +34,23 @@
 <%- end -%>
 ## How It Works [[sort]]
 
-<%= component_sections(component) %>
+### Checkpointing
+
+Vector checkpoints the journal position after every successful read. This
```

Comment:
The journald source actually checkpoints the journal position after every *batch* of successful reads. The batching is controlled by the `batch_size` option, and is done this way to avoid file churn when there are large numbers of logs coming through at once.